Tests/Editor/GameLiftPlugin/AnywherePageTests.cs

```diff
@@ -0,0 +1,176 @@
+            
+            //Assert
+            _gameLiftWrapperMock.Verify(wrapper => wrapper.CreateFleet(It.IsAny<CreateFleetRequest>()), Times.Once);
+            _coreApi.Verify(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>()), Times.Exactly(2));
+            
+            Assert.IsTrue(createFleetResult);
+        }
+        
+        [Test]
+        public void CreateAnywhereFleet_NullWrapper_ExpectFalse()
+        {
+            //Arrange
+            var gameLiftRequestAdapter = ArrangeAnywhereFleetHappyPath();
+            
+            var gameLiftPlugin = new GameLiftPlugin(_awsCredentialsFactory.Object, _coreApi.Object, _amazonGameLiftClientFactory.Object);
+            gameLiftPlugin.CurrentState = new GameLiftPlugin.State() { SelectedProfile = "unitTest" };
+            gameLiftRequestAdapter = new GameLiftRequestAdapter(gameLiftPlugin);
+
+            //Act
+            var createFleetResult =  gameLiftRequestAdapter.CreateAnywhereFleet("test").GetAwaiter().GetResult();
+            
+            //Assert
+            Assert.IsFalse(createFleetResult);
+        }
+        
+        [Test]
+        public void CreateAnywhereFleet_NullFleetName_ExpectFalse()
+        {
+            //Arrange
+            var gameLiftRequestAdapter = ArrangeAnywhereFleetHappyPath();
+
+            //Act
+            var createFleetResult =  gameLiftRequestAdapter.CreateAnywhereFleet(null).GetAwaiter().GetResult();
+            
+            //Assert
+            _gameLiftWrapperMock.Verify(wrapper => wrapper.CreateFleet(It.IsAny<CreateFleetRequest>()), Times.Once);
+            _coreApi.Verify(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>()), Times.Exactly(2));
+            
+            Assert.False(createFleetResult);
+        }
+        
+        [Test]
+        public void CreateAnywhereFleet_NullFleetId_ExpectFalse()
+        {
+            //Arrange
+            var gameLiftRequestAdapter = ArrangeAnywhereFleetHappyPath();
+            
+            _gameLiftWrapperMock.Setup(wrapper => wrapper.CreateFleet(It.IsAny<CreateFleetRequest>())).Returns(Task.FromResult(
+                new CreateFleetResponse()
+                {
+                    FleetAttributes = new FleetAttributes() { FleetId = null }, 
+                    LocationStates = new List<LocationState>()
+                }));
+
+            //Act
+            var createFleetResult =  gameLiftRequestAdapter.CreateAnywhereFleet("test").GetAwaiter().GetResult();
+            
+            //Assert
+            _gameLiftWrapperMock.Verify(wrapper => wrapper.CreateFleet(It.IsAny<CreateFleetRequest>()), Times.Once);
+            _coreApi.Verify(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>()), Times.Exactly(2));
+            
+            Assert.False(createFleetResult);
+        }
+
+        [Test] 
+        public void CreateCustomLocationIfNotExists_ThrowErrorOnListLocation_ExpectFalse()
```

Comment:
The naming convention of our package looks to be `<method>_When<state>_<result>`. This method is where I think this convention becomes most relevant as "ThrowErrorOnListLocation" sounds like a result as opposed to a state of the test and this convention helps to clarify